qtpy/QtDBus.py

```diff
@@ -4,16 +4,22 @@
 # Licensed under the terms of the MIT License
 # (see LICENSE.txt for details)
 # -----------------------------------------------------------------------------
+
 """Provides QtDBus classes and functions."""
 
-# Local imports
-from . import PYQT5, PYQT6, PYSIDE2, PYSIDE6, PythonQtError
+import sys
+from . import PYQT5, PYQT6, PYSIDE2, PYSIDE6, QtBindingsNotFoundError, QtBindingMissingModuleError, QtModuleNotInOSError
```

Comment:
* 79 width.
* Also, please a blank line between the standard library imports (i.e. `sys`) and the local ones.